app/packages/relay/src/queries/__generated__/datasetQuery.graphql.ts

```diff
@@ -88,7 +88,7 @@ export type datasetQuery$data = {
       readonly slug: string | null;
     } | null;
     readonly " $fragmentSpreads": FragmentRefs<"datasetFragment">;
-  } | null;
+  };
```

Comment:
is this safe? Could no dataset due to no permissions cause problems?

Reply:
This shouldn't be changing. I need to figure out why generated output is different. Thanks!